staging/src/k8s.io/component-helpers/storage/volume/helpers.go

```diff
@@ -16,7 +16,14 @@ limitations under the License.
 
 package volume
 
-import v1 "k8s.io/api/core/v1"
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/component-helpers/scheduling/corev1"
+	"k8s.io/klog/v2"
```

Comment:
logging inside a helper function is surprising... shouldn't the caller decide whether to log or not?

Reply:
This particular log line is at level 10. I am ok removing it.

Reply:
+1